packages/keystone/apolloErrorFormatter.js

```diff
@@ -96,130 +97,222 @@ function _patchKnownErrorCases (error, result) {
     }
 }
 
-function _extractInnerGQLError (error) {
-    const innerErrors = get(error, 'errors', [])
-    return innerErrors.find(err => get(err, 'name') === 'GQLError')
+class NonError extends Error {
```

Comment:
Strange naming :)

Reply:
Yes it's for cases if someone trying to use error formatter function for non errors

Reply:
Can you explain the case? What it error formatting for non-errors?

Reply:
I've add some docs